application/features/bootstrap/NotificationContext.php

```diff
@@ -94,7 +94,7 @@ public function aEmailIsSentToTheUsersHrContact($subject)
         Assert::assertNotEmpty($email, "No email was found with the subject: " . $subject);
 
         $user = $this->idStore->getActiveUser($this->tempEmployeeId);
-        Assert::assertContains(
```

Comment:
Was this the only assertContains in the project?

Reply:
A quick search says the answer is "yes, that's the only one".